### ISSUE TITLE
How to only use for InfluxDB

### ISSUE THREAD

**Issue by timo12357** [sremes/oomi#5]
I would like to strip all Google related dependencies and only use the software to pull consumption data to InfluxDB in OpenHAB. How to do that?

**Comment by sremes** [comment]
Did you check if the data pulling was still working? I had issues sometime ago, that the excel you get from the oomi website is no longer working, or only returns an empty sheet.

**Comment by timo12357** [comment]
I can download the csv file manually from Oomis online service, yes. I have not been able to your software to work in Armbian yet, though. The install documentation is quite brief. First hurdle is installing pandas, which is not availabe via apt.

**Comment by sremes** [comment]
Yep, previously you used to get an Excel (xlsx) file, not CSV, so I guess this is not unfortunately working currently. I need to figure out how to get the CSV file out of their system. Anyway, I would recommend installing all python deps with pip into a venv.

**Comment by timo12357** [comment]
Any progress on this?
